src/screens/Topics/ListsTopic.js

```diff
@@ -15,46 +15,35 @@ const styles = StyleSheet.create({
     marginBottom: 10,
     alignItems: 'center'
   }),
-    textTopicNotActive: (isActive) => ({
+  textTopicNotActive: (isActive) => ({
     fontFamily: 'Inter',
     fontStyle: 'normal',
     fontWeight: '500',
     fontSize: 12,
-    color:  isActive ?  colors.white : colors.mine_shaft
+    color: isActive ? colors.white : colors.mine_shaft
     // paddingLeft: 5,
-  }),
-})
+  })
+});
 
 const ListTopics = ({item, i, onPress}) => {
-    const [isActive, setIsActive] = React.useState(false)
+  const [isActive, setIsActive] = React.useState(false);
 
-    const handleSelectedLanguage = (id) => {
-        setIsActive((prevState) => !prevState)
-        if(onPress) {
-            onPress(id)
-        }
+  const handleSelectedLanguage = (id) => {
+    setIsActive((prevState) => !prevState);
+    if (onPress) {
+      onPress(id);
     }
+  };
 
-    return (
+  return (
     <Pressable
-    onPress={() =>
-      handleSelectedLanguage(item.topic_id)
-    }
-    key={i}
-    style={
-      [styles.bgTopicSelectNotActive(isActive)]
-    }
-    >
-    <Text>{item.icon}</Text>
-    <Text
-      style={
-        [styles.textTopicNotActive (isActive)]
-      }>#{item.name}</Text>
-  </Pressable>
-    )
-
-}
-
-
-
-export default React.memo(ListTopics)
\ No newline at end of file
+      onPress={() => handleSelectedLanguage(item.topic_id)}
+      key={i}
+      style={[styles.bgTopicSelectNotActive(isActive)]}>
+      <Text>{item.icon}</Text>
+      <Text style={[styles.textTopicNotActive(isActive)]}>#{item.name}</Text>
+    </Pressable>
+  );
+};
+
+export default React.memo(ListTopics);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [18-50]

The `handleSelectedLanguage` function toggles the `isActive` state every time it's called. This might not be the expected behavior if this function is meant to handle language selection. If a language is selected, it should probably remain active until another language is selected. Consider revising this logic to ensure that the active state accurately reflects the user's selections.
```diff
  const handleSelectedLanguage = (id) => {
-   setIsActive((prevState) => !prevState);
+   setIsActive(true);
    if (onPress) {
      onPress(id);
     }
  };
```
Also, there is no newline at the end of file on line 40. It's a common convention to end files with a newline because some tools require it to function properly or to prevent them from adding one in automatically and marking the file as changed. Please add a newline at the end of the file.
```diff
export default React.memo(ListTopics);
+ 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->